packages/tre/src/storage/sqlite.ts

```diff
@@ -205,4 +205,8 @@ export class SqliteStorage extends LocalStorage {
       sql`SELECT key, attributes FROM ${unsafe_raw(this.namespace)}`
     );
   }
+
+  getSqliteDatabase(): DatabaseType {
+    return this.database;
```

Comment:
I almost wonder if this should be a different database. You probably shouldn't be able to inspect your KV data through D1, for instance. Although, if storage is changing soon, that's not a big deal or a blocker for this PR